example.py

```diff
@@ -4,8 +4,16 @@
 url = os.environ.get('MIQURL') or 'http://localhost:3000/api'
 username = os.environ.get('MIQUSERNAME') or 'admin'
 password = os.environ.get('MIQPASSWORD') or 'smartvm'
+token = { 'token': os.environ.get('MIQTOKEN') }
 
-client = MiqApi(url, (username, password))
+client = None
+
+if os.environ.get('MIQTOKEN'):
+    print("\nAuthenticating with the API token")
+    client = MiqApi(url, token)
+else:
+    print("\nAuthenticating with the user credentials")
+    client = MiqApi(url, (username, password))
```

Comment:
and here I'd  use  `dict(user=username, password=password)`
format for consistency